Adding tests for login and register

dhillonks opened the pull request:
Added tests for the following:
- `src/components/SLogin`
- `src/components/Register`

For mocking the requests, I used the already existing `src/testServer.js` module to handle the `login` and `signup` requests.

The tests can be found in:
```
src/components/accounts/__tests__/SLogin.test.js
src/components/accounts/__tests__/Register.test.js
```
![image](https://user-images.githubusercontent.com/52294825/142363164-e58bd09d-804e-4ff3-a5c8-68c3f313cdd1.png)

redxzeta reviewed (changes requested):
I think you could just put fake supabase credentials so it could look like this
```
const SUPABASE_URL = "https://paw-dev.supabase.co";
const KEY = "pawternityhubo";
const supabase = createClient(SUPABASE_URL, KEY);
test("should register sucessfully", async () => {
  render(
    <Provider value={supabase}>
      <Register />
    </Provider>
  );

```

and just change the mock api

can you make the branch based off on `paw-dev` and pr to that branch

redxzeta commented on src/testServer.js:
```diff
@@ -25,11 +26,48 @@ const server = setupServer(
+  server.resetHandlers();
+});
+
+const SUPABASE_URL = "https://vuxohlizzevgcsxlrggf.supabase.co";
```
Was wondering if you could put a dummy supabase url and key

dhillonks replied:
done!

dhillonks commented:
> I think you could just put fake supabase credentials so it could look like this
> 
> ```
> const SUPABASE_URL = "https://paw-dev.supabase.co";
> const KEY = "pawternityhubo";
> const supabase = createClient(SUPABASE_URL, KEY);
> test("should register sucessfully", async () => {
>   render(
>     <Provider value={supabase}>
>       <Register />
>     </Provider>
>   );
> ```
> 
> and just change the mock api
> 
> can you make the branch based off on `paw-dev` and pr to that branch

Added a supabase client for the test runs, looks like it passed the CI/CD through github now. Also changed the target branch to `paw-dev`. Could you re-review?
